bokehjs/src/less/buttons.less

```diff
@@ -92,6 +92,14 @@
     .button-variant(@btn-danger-color; @btn-danger-bg; @btn-danger-border);
   }
 
+  .bk-btn-light {
+    border-width: 0px;
```

Comment:
Setting border width will affect layout (both CSS and bokeh's) and make components not align (subtly, but still). One should keep the width intact and set `border-color` to `transparent`.

Reply:
I've pushed a fix.